src/persist-client/src/cache.rs

```diff
@@ -218,6 +218,7 @@ impl PersistClientCache {
 /// rtt latency task, there's the possibility for it being confusing at some
 /// point. Err on the side of more data (including the latency measurements) to
 /// start.
+#[allow(clippy::unused_async)]
```

Comment:
Is there a reason to keep these async?

Reply:
I imagine they are `async` now so people remember to call them from tokio runtime threads, or because we may add async work in the future

Reply:
:point_up: the former. persist usually has a comment when we're doing that, but looks like it got missed on these